RIOT-OS/RIOT
# Kconfig: Expose gnrc/ipv6/ext/frag configurations

### Contribution description
This PR moves the configuration macros of `gnrc_ipv6_ext_frag` to the `CONFIG_` namespace, and exposes them to Kconfig.

In the test application a check was added, so the parameters are not changed via CFLAGS when Kconfig is being used, ~that's why this depends on #12913.~

### Testing procedure
- `tests/gnrc_ipv6_ext_frag` should still work as usual.
- You should be able to configure the module's parameters via `make menuconfig` in the test application.

### Issues/PRs references
Part of #12888
~Depends on #12913~
